test/tools/verify/cmd/main.go

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 const (
-	machineReadyCheckPeriod  = 2 * time.Second
-	machineReadyCheckTimeout = 5 * time.Minute
+	machineReadyCheckPeriod  = 5 * time.Second
+	machineReadyCheckTimeout = 10 * time.Minute
```

Comment:
I have increased timeous, as the verify tool failed when more that one machine was being created at the same time